### ISSUE TITLE
Scalable and multi size icons

### ISSUE THREAD

**Issue by sebastinas** [pwmt/zathura#72]
*On GitLab by @sanjay_ankur on Aug 9, 2018, 16:10*

---

Would the svg version of the icon be available someplace? I could generate icons of different sizes so that they could be used with different screen resolutions. Currently, sometimes I don't get an icon on my bigger screens (and I assume the issue is that there is a single size icon available for zathura only):

```
$ rpmls zathura | grep icon
-rw-r--r--  /usr/share/icons/hicolor/128x128/apps/org.pwmt.zathura.png
```

**Comment by sebastinas** [comment]
mentioned in commit 48469dd0ef0b3909e27fb4bbb963a6de5e886057

**Comment by sebastinas** [comment]
Fixed in 48469dd0.

**Comment by sebastinas** [comment]
closed
